sdk/healthdataaiservices/azure-health-deidentification/CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Release History
 
+## 1.0.0 (2024-12-13)
```

Comment:
What is the plan for this GA version? Our paging with continuationToken is not supported yet, https://github.com/Azure/autorest.typescript/issues/3041. We need to priorize with the feature ask.